isl_test.c

```diff
@@ -6048,6 +6075,8 @@ int test_aff(isl_ctx *ctx)
 	isl_aff *aff;
 	int zero, equal;
 
+	if (test_pwa(ctx) < 0)
```

Comment:
I'd call this function `test_pa` for consistency with the `test_upa` below

Reply:
Done.